src/frontend/utils/conversejs/converse-plugins/chatPlugin.ts

```diff
@@ -0,0 +1,33 @@
+import { MessageType, XMPP } from 'types/XMPP';
+import { converse } from './../../window';
+
+const PLUGIN_NAME = 'chat-plugin';
+
+const addChatPlugin = (xmpp: XMPP) =>
+  converse.plugins.add(PLUGIN_NAME, {
+    dependencies: ['converse-muc'],
+    initialize() {
+      const _converse = this._converse;
+
+      const sendMessageWithConverse = (message: string): void => {
```

Comment:
```suggestion
      const sendMessage = (message: string): void => {
```

`withConverse` is useless. This is a plugin in converse, `converse.sendMessageWithConverse` is redundant don't you think ?

Reply:
Yes i agree, it makes no sense (I think I named it like this in the beginning, to make it clearer, and then forgot to changed it).